### PR TITLE
Add minimum_allowed_balance to Account schema (RILP-263)

### PR DESCRIPTION
**Pull request by MatthewPhinney**

### CODE REVIEW

File: schemas/Account.json [hunk 1]
```diff
@@ -44,6 +44,14 @@
       "description": "A fingerprint of the account's client certificate",
       "type": "string",
       "pattern": "^[0-9A-F]{2}(:[0-9A-F]{2}){19,127}$"
+    },
+    "minimum_allowed_balance": {
+      "description": "The minimum balance permitted on this account",
+      "oneOf": [{
+        "$ref": "Amount.json"
+      }, {
+        "enum": ["infinite"]
```
Comment (clark800) [inline schemas/Account.json]: i think we would want "-infinity"

**Comment by MatthewPhinney** [comment]
Do we plan to have a `maximum_allowed_balance`?

**Comment by clark800** [comment]
I don't think there is a task for that yet, so for now I think it's fine to just add `minimum_allowed_balance`.

**Comment by clark800** [comment]
LGTM

**Pull request merged**